test/mongoose_push_web/controllers/api_v3_notification_controller_test.exs

```diff
@@ -40,7 +40,7 @@ defmodule MongoosePushWeb.APIv3NotificationControllerTest do
         Jason.encode!(Map.drop(ControllersHelper.alert_request(), ["alert"]))
```

Comment:
How about build a body in a separate variable and then use it?
```elixir
body = 
    ControllersHelper.alert_request()
    |> Map.drop(["alert"])
    |> Jason.encode!()
```

Reply:
The matter of preference. To be considered when doing property-based stuff :)

Reply:
This is a small change, you can just copy-paste the code from a snippet and also it is a perfect case for using `|>` operator as shown here: https://elixirschool.com/en/lessons/basics/pipe-operator/

Reply:
Alek, don't get me wrong, I do know what the pipe operator is. Since I misunderstood a bit my current story purpose (that actually we aim to keep already existing test scenarios), I will incorporate your suggestions as this code won't disappear right after I do that.